tests/core/pyspec/eth2spec/test/context.py

```diff
@@ -106,6 +109,18 @@ def default_balances(spec: Spec):
     return [spec.MAX_EFFECTIVE_BALANCE] * num_validators
 
 
+def default_balances_electra(spec: Spec):
+    """
+    Helper method to create a series of default balances for Electra.
+    Usage: `@with_custom_state(balances_fn=default_balances_electra, ...)`
+    """
+    if not is_post_electra(spec):
+        return default_balances(spec)
```

Comment:
ditto. Is it possible that instead of adding new `default_balances_electra,` we modify `default_balances` by adding new `if is_post_electra(spec)` logic?

Reply:
I was thinking about this approach but then decided to keep default balances as is because it makes a test run with 32 ETH balance validators which is also important for Electra. Ideally, we should run tests with `MIN_ACTIVATION_BALANCE`, `MAX_EFFECTIVE_BALANCE_ELECTRA` and something in between. It is probably better to handle this modification separately.

Reply:
It's probably more just the naming problem; when I read `default_balances_electra`, I thought it is *just* "the default balances post-electra," but it is actually compatible with pre-electra. But I understand what you wanted to present.

Reply:
what about calling it `max_balances` instead? To me it explains pretty clearly what it does